src/officehours_api/views.py

```diff
@@ -184,7 +184,10 @@ def post(self, request, pk):
             m.start()
         except DisabledBackendException as e:
             return Response({'Start Meeting': e.message}, status=status.HTTP_400_BAD_REQUEST)
-        m.save()
+        try:
+            m.save()
+        except Exception as e:
+            return Response({'Start Meeting': e.message}, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
I worked with Divyanga on this. I believe this try/except needs to be in the `MeetingDetail` view, since that's where the `put` for meetings happens (which is where the error originates).

Here are the branches with some of our work, including unit tests:
- with request factory: https://github.com/tl-its-umich-edu/remote-office-hours-queue/commit/8d7bfb410955d0585e670e9e56a53d41c1093b54
- with Django client: https://github.com/tl-its-umich-edu/remote-office-hours-queue/commit/781cdf936627dbe141b62fa03973d8d93d7943d6

Reply:
Hi, myself and my partner (@garima-shah) are looking to run these tests in order to resolve another open issue. We were wondering how to go about that?

Reply:
Hi, @challasr, you run tests by using the command documented here: https://docs.djangoproject.com/en/4.2/topics/testing/overview/#running-tests

Since the project uses Docker, you'll have to do something like this:
```
docker-compose run --entrypoint="" web python manage.py test
```
Or find the name of the container, enter it using `docker exec`, and run the `python manage.py test` command directly.